index.Rmd

```diff
@@ -53,12 +57,12 @@ For each of these problems, we map them to the type of data analysis question be
 
 | Question type | Description | Example |
 |---------------|-------------|---------|
-| Descriptive | A question which asks about summarized characteristics of a data set without interpretation (i.e., report a fact). | How many people live in each US state? |
-| Exploratory | A question asks if there are patterns, trends, or relationships within a single data set. Often used to propose hypotheses for future study. | Does politcal party voting change with indicators of wealth in a set of data collected from groups of individuals from several regions in the United States? |
-| Inferential | A question that looks for patterns, trends, or relationships in a single data set **and** also asks for quantification of how applicable these findings are to the wider population. | Does politcal party voting change with indicators of wealth in the United States? |
-| Predictive | A question that asks about predicting measurements or labels for individuals (people or things). The focus is on what things predict some outcome, but not what causes the outcome. | What political party will someone vote for in the next US election? |
-| Causal | A question that asks about whether changing one factor will lead to a change in another factor, on average, in the wider population. | Does wealth lead to voting for a certain political party candidate in the US Presidential election? |
-| Mechanistic | A question that asks about the underlying mechanism of the observed patterns, trends, or relationship (i.e., how does it happen?) | How does wealth lead to voting for a certain political party candidate in the US Presidential election? |
+| Descriptive | A question which asks about summarized characteristics of a data set without interpretation (i.e., report a fact). | How many people live in each province or territory in Canada? |
+| Exploratory | A question asks if there are patterns, trends, or relationships within a single data set. Often used to propose hypotheses for future study. | Does political party voting change with indicators of wealth in a set of data collected from groups of individuals from several regions in Canada? |
+| Inferential | A question that looks for patterns, trends, or relationships in a single data set **and** also asks for quantification of how applicable these findings are to the wider population. | Does political party voting change with indicators of wealth in Canada? |
```

Comment:
For the questions in the table, maybe we can differentiate the "Inferential" a little more by indicating that this question is referring to a population. And that way we can contrast an exploratory and inferential question by making it essentially the same question but changing the group it's referring to, something like: "Does political party voting change with indicators of wealth for every single person in Canada?" (or something along those lines) and then exploratory is the same question but just about the data

Reply:
Fixed in 6fdf3cf